modules/services/redshift-gammastep/lib/options.nix

```
@@ -36,7 +69,7 @@ in {
     };
 
     longitude = mkOption {
-      type = types.nullOr types.str;
+      type = types.nullOr types.float;
```

Could do
```suggestion
      type = with types; nullOr (either str float);
      apply = toString;
```
to help backwards compatibility.